plugins/UFM_NDT_Plugin/ndt_file_creator.py

```diff
@@ -61,6 +61,11 @@ def allocate_request_args(parser, release_version="1.0"):
                          required=None, default="no",
                          choices=["no", "yes"],
                          help="Flag if to include in NDT file currently disconnected Switch ports. Default - no")
+    request.add_argument("-e", "--include_error_ports", action="store",
```

Comment:
flags should not have yes/no choice. it's redundant. (relevant also for `include_down_ports`)
using `store_true` is better.
```python
request.add_argument("-e", "--include_error_ports", action="store_true",
    required=None,
    help=...
```

Reply:
fixed